src/instructlab/cli/process/remove.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.defaults import ILAB_PROCESS_STATUS
+from instructlab.process.process import (
+    display_remove_list,
+    filter_process_with_conditions,
+    remove_process,
+)
+
+
+@click.command(name="remove")
+@click.argument("process_uuid", required=False)
```

Comment:
So what's the difference between `remove` and `prune` if both act on multiple processes and support filtering by `--older` and `--force`? Should these have different goals? (Prune to batch clean up by filter; remove to require a ID to pass.)